content/news/XXXX-XX-XX-mixxx-2-3-6-released.md

```diff
@@ -0,0 +1,75 @@
+title: Mixxx 2.3.6 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.6, release announcement
+status: draft
+comments: yes
+
+Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.
+
+This little bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user and ensuring a reliable mixing experience.
```

Comment:
```suggestion
This small bug fix release may appear insignificant, but it manifests our effort to deliver rock-solid quality and ensure a reliable mixing experience.
```
'little' sounds a bit like 'cute'

Reply:
I agree with the `s/little/small` and the `s/ensuring/to ensure/` but I prefer the "testament" version of the rest of the sentence. Alternatively
```suggestion
This small bug fix release may appear insignificant, but it serves as evidence of the rock-solid quality we bring to every user and to ensure a reliable mixing experience.
```